sdk/tests/merkleTree.test.ts

```
@@ -1,31 +1,35 @@
-import { strict as assert } from 'node:assert';
+import { strict as assert } from "node:assert";
 
-import * as crypto from 'crypto';
+import * as crypto from "crypto";
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)